packages/wonder-blocks-dropdown/components/single-select-menu.md

```diff
@@ -0,0 +1,227 @@
+The single select menu allows the selection of one item. Clients are
+responsible for keeping track of the selected item in the menu.
+
+TODO(sophie): keyboard navigation and dismissal
+
+The first menu has a starting placeholder and a set width.
+
+The second menu starts with a starting selected item. One of the items is
+disabled and thus cannot be selected.
+
+The third menu is disabled.
+
+```js
+const React = require("react");
+const {View} = require("@khanacademy/wonder-blocks-core");
+const {StyleSheet} = require("aphrodite");
+
+const styles = StyleSheet.create({
+    row: {
+        flexDirection: "row",
+        height: 180,
+    },
+    strutLike: {
+        width: 8,
+    }
+});
+
+class ExampleWithPlaceholder extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            selectedValue: null,
+        }
+    }
+
+    handleSelected(selected) {
+        console.log(`${selected} was selected!`);
+        this.setState({
+            selectedValue: selected,
+        });
+    }
+
+    render() {
+        return <SingleSelectMenu
+            items={[
+                {
+                    label: "Banana wrapped in a peel",
+                    value: "banana"
+                },
+                {
+                    label: "Apple",
+                    value: "apple"
+                },
+                {
+                    label: "Grape",
+                    value: "grape"
+                },
+            ]}
+            light={false}
+            onChange={(selected) => this.handleSelected(selected)}
```

Comment:
super nit: `this.handleChange(selected)` as per the https://github.com/Khan/style-guides/blob/master/style/react.md#name-handlers-handleeventname.